src/tests/Dashboard/reducer/conceptReducer.test.js

```diff
@@ -13,8 +13,10 @@ import {
   ADD_NEW_SET_ROW,
   REMOVE_SELECTED_SET,
   ADD_SELECTED_SETS,
-  PRE_POPULATE_SETS, UNPOPULATE_PRE_POPULATED_SETS,
+  PRE_POPULATE_SETS,
```

Comment:
Is this related to increasing the test coverage?

Reply:
Its a refactor of the imports for consistency and readability- each on its own line.